### DISCUSSION TITLE
Student developer pack not received

### DISCUSSION THREAD

**Discussion by Ignikhilyt** [community/community#147900] [GitHub Education]
It's been more than 4 days and still I haven't recived it please help me

**Comment by davevad93** [comment]
Hi @Ignikhilyt ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Ignikhilyt was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Ignikhilyt** [reply]
How much I should wait now give me a last deadline so I can be sure

**Reply by davevad93** [reply]
Hey @Ignikhilyt I can't give you a sure deadline because the waiting time isn't the same for everyone, although if you were approved 4 days ago, I guess by next week you should have them. 👍

**Reply by Ignikhilyt** [reply]
Ahhh 😫
